client/src/components/Create.jsx

```diff
@@ -41,6 +44,7 @@ class Create extends React.Component {
   }
 
   handleSubmit(event) {
+    var that = this;
```

Comment:
I think you can avoid setting `that` if you use arrow functions in the axios success callback?